src/gtimelog/timelog.py

```diff
@@ -17,9 +17,6 @@
 from operator import itemgetter
```

Comment:
This is the right file with the superfluous `import sys` at the top.

(What a coincidence, it's on line 13 on both files, or I would've noticed sooner!)